packages/tailwindcss-language-server/src/project-locator.ts

```diff
@@ -652,7 +652,9 @@ async function* detectContentFiles(
       yield normalizePath(file)
     }
 
-    for (let { base, pattern } of result.globs) {
+    for (let { base, pattern, negated } of result.globs) {
+      if (negated) continue
+
```

Comment:
I think this change is not required.
```suggestion
    for (let { base, pattern } of result.globs) {

```